js/bundle/src/encoder.ts

```diff
@@ -23,8 +30,8 @@ export class BundleBuilder {
   private currentResponsesOffset = 0;
   private compatAdapter: CompatAdapter;
 
-  constructor(formatVersion: string = 'b2') {
-    if (formatVersion != 'b1' && formatVersion != 'b2') {
+  constructor(formatVersion: FormatVersion = DEFAULT_VERSION) {
```

Comment:
nit: Also use `FormatVersion` throughout the rest of this file, e.g. in `createCompatAdapter` and in the `formatVersion` field of the `CompatAdatper` implementations.